examples/angular2/package.json

```diff
@@ -1,18 +1,19 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "cp node_modules/systemjs/node_modules/es6-module-loader/dist/es6-module-loader.js node_modules/systemjs/dist && ./node_modules/.bin/tsd query angular2 node-uuid storejs node --action install",
-    "dev": "tsc --watch -m commonjs -t es5 --emitDecoratorMetadata app.ts"
+    "postinstall": "./node_modules/.bin/tsd query node-uuid-base node-uuid-cjs storejs --action install",
+    "dev": "tsc --watch"
   },
   "devDependencies": {
     "tsd": "^0.6.0",
-    "typescript": "^1.5.0-beta"
+    "typescript": "^1.7.3"
   },
   "dependencies": {
-    "bower-traceur-runtime": "git://github.com/jmcriffey/bower-traceur-runtime#0.0.87",
+    "angular2": "2.0.0-beta.0",
+    "systemjs": "0.19.6",
+    "rxjs": "5.0.0-beta.0",
```

Comment:
this peer dep issue seems to be something wrong in angular based on what you said